pyproject.toml

```diff
@@ -70,7 +70,7 @@ testing = [
     "requests>=2.20.0",
 ]
 quality = [
-    "flake8>=3.9.0",
+    "ruff>=0.0.260",
```

Comment:
out of security, I would have added an upper bound as well 0.1.0 for instance to avoid troubles with the CI later but nothing major :+1: